### PR TITLE
Update versions.json

### PR DESCRIPTION
**Pull request by samgnaniah**
Update 7.0.0 URL in versions page.

### CODE REVIEW

File: en/docs/assets/versions.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "notes": "7.1.0/micro-integrator/overview/about-this-release-7.1.0"
     },
     "7.0.0": {
-        "doc": "7.0.0",
+        "doc": " https://ei.docs.wso2.com/en/7.0.0/",
         "notes": "7.0.0/micro-integrator/overview/about-this-release-7.0.0"
```
Comment (CharukaK) [inline en/docs/assets/versions.json:15]: We have to do the same thing for the about the release link too

File: en/docs/assets/versions.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
         "notes": "7.1.0/micro-integrator/overview/about-this-release-7.1.0"
     },
     "7.0.0": {
-        "doc": "7.0.0",
+        "doc": " https://ei.docs.wso2.com/en/7.0.0/",
```
Comment (CharukaK) [inline en/docs/assets/versions.json:14]: remove the space at the beginning

File: en/docs/assets/versions.json (hunk 1 above)
Comment (samgnaniah) [reply en/docs/assets/versions.json]: Fixed.

File: en/docs/assets/versions.json (hunk 2 above)
Comment (samgnaniah) [reply en/docs/assets/versions.json]: Fixed.

**Pull request merged**